release-notes/1.20.2.md

```diff
@@ -0,0 +1,24 @@
+## Introduction
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.20.2.
+
+Please see [component versions](https://anbox-cloud.io/docs/ref/component-versions) for a list of updated components.
+
+## New features & improvements
+* VPU slots for an app are synchronised through the Anbox Application Registry (AAR).<!--AC-2143-->
+* Extended logging is available for WebRTC connections. This extended logging includes logging of local and remote ICE candidates which can be used by enabling the feature flag `webrtc.enable_ice_logging`.<!--AC-2184-->
+* Android security updates for January 2024 (see [Android Security Bulletin - January 2024](https://source.android.com/docs/security/bulletin/2024-01-01) for more information).<!--AC-2203-->
+* The Android WebView has been updated to [120.0.6099.193](https://chromereleases.googleblog.com/2024/01/chrome-for-android-update.html).
+
+## Bugs fixed
+* In the Anbox Management Service (AMS), application is created successfully even when the `post-stop` hook exits with a non-zero status.<!--AC-2167-->
+* Automotive audio control service (`android.hardware.automotive.audiocontrol`) gets launched by accident when running the Anbox Cloud AOSP image.<!--AC-2199-->
+* LXD error `NonReusableSucceeded` is not handled properly. <!--AC-2212-->
+* System security updates for instances are applied even when the corresponding AMS configuration was set to explicitly disable them. <!--AC-2122-->
+* With all Android versions, screen recording from the quick settings fails to start.<!--AC-2180-->
+* Starting an Android 11 instance without a GPU fails to start. <!--AC-2181-->
+* On the web dashboard, the *Create session* button is clickable even when the *Application* field is empty.<!--AC-2179-->
+* [LP #2046348](https://bugs.launchpad.net/anbox-cloud/+bug/2046348) When dynamically changing the screen resolution upon joining a streaming session to a lower resolution than the configured resolution, the resolution of the resulting video stream does not match. Instead, it stays at the resolution initially configured for the streaming session.<!--AC-2188-->
+* When running WebGL based applications, the WebView shows an error and indicates that the WebView APK shipped as part of the Android image does not support WebGL.<!--AC-2214-->
```

Comment:
It doesn't look like we have a public bug for this (we should though). Can we reference https://discourse.ubuntu.com/t/enable-webgl-in-anbox-cloud/41170/4 instead?